docs/dev-guide/test-quality-checklist.md

```
@@ -0,0 +1,60 @@
+scope: internal-development-strategy
+---
+
+# Test Quality Checklist
+
+> **Scope**: This checklist applies to ALL test PRs in defense-in-depth,
+> regardless of author (Human, Main Agent, or Jules). It is an internal
+> development standard, not a requirement for DiD package users.
+
+## PR Review Checklist
+
+### Assertion Quality
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)